features/google/src/i18n.js

```diff
@@ -0,0 +1 @@
+export { default } from "!silly-i18n";
```

Comment:
Why was this file added back?

Reply:
This was coming from the `main` branch, after merging it into my branch.

Reply:
I don't understand why this happened, I just merged `main` into my branch 😕 

@g3o314 had a problem with the i18n in Windows (her solution here: https://github.com/polypoly-eu/polyPod/pull/918 ). Do you think this is related? Does her PR solve this or should I do what you're saying - removing the file and restoring the changes?

Reply:
I don't think that #918 is related, since even after the changes here we ultimately still import `!silly-i18n` just not directly in at least one case. I suggest to remove the `i18n.js` module here, and replace any import of it with `import i18n from "!silly-i18n"`.

Reply:
Ok, I was asking just in case.

No problem, I'll do that right now (in a different PR).